### PR TITLE
Fix message formatting when max scan count is exceeded.

### PR DESCRIPTION
**Pull request by monterinio**

### CODE REVIEW

File: src/main/java/kafdrop/service/KafkaMonitorImpl.java [hunk 1]
```diff
@@ -280,10 +280,10 @@ public SearchResultsVO searchMessages(String topic,
         case EXCEEDED_MAX_SCAN_COUNT:
           results.setCompletionDetails(
             String.format(
-              "Search timed out after scanning %d messages.  Last scanned message timestamp was %d.  Adjust your time" +
-                " span for more results.",
+              "Search timed out after scanning %d messages. Last scanned message timestamp was %s. Adjust your time" +
+              " span for more results.",
```
Comment (Bert-R) [inline src/main/java/kafdrop/service/KafkaMonitorImpl.java]: Can't we just fix the format?
```suggestion
              "Search timed out after scanning %1$d messages. Last scanned message timestamp was %2$tF %2$tT. Adjust your time" +
                " span for more results.",
```

File: src/main/java/kafdrop/service/KafkaMonitorImpl.java [hunk 2]
```diff
@@ -280,10 +280,10 @@ public SearchResultsVO searchMessages(String topic,
...
               records.getMessagesScannedCount(),
-              records.getFinalMessageTimestamp()));
+              records.finalMessageTimestampFormatted()));
```
Comment (Bert-R) [inline src/main/java/kafdrop/service/KafkaMonitorImpl.java]:
```suggestion
              records.getFinalMessageTimestamp()));
```

**Review by Bert-R — CHANGES_REQUESTED** [review]
Thanks for the contribution!
Added a proposal for a simpler fix.

**Comment by monterinio** [comment]
I will adjust as requested today's evening! Thanks.

File: src/main/java/kafdrop/service/KafkaMonitorImpl.java [hunk 3]
```diff
@@ -280,8 +280,8 @@ public SearchResultsVO searchMessages(String topic,
         case EXCEEDED_MAX_SCAN_COUNT:
           results.setCompletionDetails(
             String.format(
-              "Search timed out after scanning %d messages.  Last scanned message timestamp was %d.  Adjust your time" +
-                " span for more results.",
+              "Search timed out after scanning %d messages. Last scanned message timestamp was %2$tF %2$tT. Adjust your time" +
+              " span for more results.",
```
Comment (Bert-R) [inline src/main/java/kafdrop/service/KafkaMonitorImpl.java]:
```suggestion
                " span for more results.",
```

**Review by Bert-R — COMMENTED** [review]
Looks good, except for a very small thing: the two spaces on the next line are accidentally removed.

File: src/main/java/kafdrop/service/KafkaMonitorImpl.java (hunk 3 above)
Comment (monterinio) [reply src/main/java/kafdrop/service/KafkaMonitorImpl.java]: Ha! had the opposite feeling that there are too many of them. (:

Comment (Bert-R) [reply src/main/java/kafdrop/service/KafkaMonitorImpl.java]: The first parameter of `String.format` is broken in two lines. The second half is indented a bit more to show it's not the second parameter but second half of the first one.

**Review by Bert-R — APPROVED** [review]
Thanks!

**Pull request merged**